lib/maplibre-compose/src/maplibreNativeMain/kotlin/dev/sargunv/maplibrecompose/compose/offline/DownloadProgress.kt

```diff
@@ -0,0 +1,19 @@
+package dev.sargunv.maplibrecompose.compose.offline
+
+public sealed interface DownloadProgress {
+  public data object Unknown : DownloadProgress
+
+  public data class Normal(
```

Comment:
```suggestion
  public data class State(
```
? `DownloadProgress.Normal` sounds odd, IMO.

Reply:
How about "Healthy"? As opposed to unknown, error, etc

Reply:
hm...